product/roundhouse/runners/RoundhouseMigrationRunner.cs

```diff
@@ -39,6 +40,7 @@ public sealed class RoundhouseMigrationRunner : IRunner
             bool silent,
             bool dropping_the_database,
             bool dont_create_the_database,
+            bool dont_alter_the_database,

```

Comment:
I see some others that could probably be cleaned up as well. I'm just asking as it is part of the config.

You can pull the setting out of `configuration.DoNotAlterDatabase`. Less values passing in is always a good thing. :)